poc/policy/policy.py

```diff
@@ -0,0 +1,48 @@
+"""
+Interface for DP policies.
+
+A DP policy relies on all entities of an aggregation protocol to add the right
+amount of noise to ensure the final aggregate result is differentially private.
+"""
+
+class DpPolicy:
+    # Client measurement type.
+    Measurement = None
+    # Aggregate share type, owned by an Aggregator.
+    AggShare = None
+    # Aggregate result type, unsharded result from all Aggregators.
+    AggResult = None
+    # Debiased aggregate result type.
+    DebiasedAggResult = None
+
+    def add_noise_to_measurement(self,
+                                 meas: Measurement,
+                                 ) -> Measurement:
+        """
+        Add noise to measurement, if required by the Client-DP mechanism.
+        """
+        raise NotImplementedError()
+
+    def add_noise_to_agg_share(self,
+                               agg_share: AggShare,
+                               meas_count: int,
+                               min_batch_size: int,
```

Comment:
Remind me why we distinguish between `meas_count` and `min_batch_size`? Also, please document these parameters in the function comment below.

If this is related to the "compensation" idea (https://docs.google.com/document/d/1a5EoDLswyI0IKv2xwyqF4kMJNqRdVKapskeqwPtuqa4/edit), then I think we agreed to hold off on that idea in the intial draft. cc/ @wangshan to confirm.

Similar question for debias_agg_result() below.